### PR TITLE
iOS web view plugin

### PR DESCRIPTION
**Pull request by toufikzitouni**
I’m using GeneratedPluginRegistrant because that’s the supported plugin
in the new Flutter version. I couldn’t compile the code without it
unfortunately

### CODE REVIEW

**Comment by lejard-h** [comment]
Thanks, I will test it, anything to configure or it just work ?

File: example/ios/Podfile.lock [hunk 1]
```diff
@@ -1,3 +1,22 @@
-PODFILE CHECKSUM: fe68fbd35e5bc75c5acdec41319edc0cdaebd038
+PODS:
```
Comment (lejard-h) [inline example/ios/Podfile.lock]: I guess this file should be in gitignore

**Comment by toufikzitouni** [comment]
No special setup needed. But I do think the plugin should be updated to work with the new version of the Flutter sdk that uses GeneratedPluginRegistrant. Were you able to test it successfully?

**Comment by lejard-h** [comment]
Yes tested successfuly, I had to update my project.

But the clearCookies params does not work, the app just shutdown for me, without error.

you can test it, 
https://github.com/dart-flitter/flitter/blob/webview_ios/lib/services/flutter_gitter_auth.dart#L61

**Comment by lejard-h** [comment]
Last thing, is it possible to have a parameter to have a completely fullscreen view, without the top bar with the "Done" button ? If you prefer we can do it later, in an other pull request it is not blocking

**Comment by toufikzitouni** [comment]
I've added the changes you requested, including adding a fullScreen param to toggle full screen mode for the WebView

**Comment by lejard-h** [comment]
https://github.com/dart-flitter/flutter_webview_plugin/issues/5
https://github.com/dart-flitter/flutter_webview_plugin/issues/4

**Pull request merged**